src/viur/core/prototypes/list.py

```diff
@@ -192,6 +207,8 @@ def list(self, *args, **kwargs) -> t.Any:
                     default_order = default_order(query)
 
                 if default_order:
+                    logging.debug(f"Applying {default_order=}")
```

Comment:
Are these debugs indented or forgotten to be removed after development?

Reply:
I would like to keep these, as they might help to find unwanted behavior faster, especially when porting projects to viur-core 3.6.

Reply:
okay